{{cookiecutter.project_slug}}/clients/mobile/react-native/package.json

```diff
@@ -5,16 +5,16 @@
   "author": "Thinknmimble",
   "main": "node_modules/expo/AppEntry.js",
   "scripts": {
-    "start": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} SENTRY_AUTH_TOKEN=${SENTRY_AUTH_TOKEN} npx expo start",
-    "start:clear": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} npx expo start --clear",
-    "start:prod": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} npx expo start --no-dev --minify",
-    "start:android": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} npx expo start --android",
-    "start:ios": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} npx expo start --ios",
-    "start:web": "source .env && SENTRY_PROJECT_NAME=${SENTRY_PROJECT_NAME} npx expo start --web",
```

Comment:
These are technically all the same, it still shows you the menu to select your device unfortunately I do not know how to create one script to use. The problem with a script is that it runs in a separate shell and that appears to disable  terminal inputs (notice how the explicit vars are declared as part of the run without commas or &&). I would assume a script would work if we could pipe the output to the command!
@oudeismetis